skbio/stats/composition.py

```diff
@@ -642,6 +645,134 @@ def ilr_inv(mat, basis=None, check=True):
     return clr_inv(np.dot(mat, basis))
 
 
+@experimental(as_of="0.5.1-dev")
+def alr(mat, denominator_idx=0):
+    r"""
+    Performs additive log ratio transformation.
+
+    This function transforms compositions from a D-part Aitchison simplex to
+    a non-isometric real space of D-1 dimensions. The argument
+    `denominator_col` defines the index of the column used as the common
+    denominator. The :math: `alr` transformed data are amenable to multivariate
+    analysis as long as statistics don't involve distances.
+
+    :math:`alr: S^D \rightarrow \mathbb{R}^{D-1}`
+
+    The alr transformation is defined as follows
+
+    .. math::
+        alr(x) = [\ln \frac{x_1}{x_D}, \ldots, \ln \frac{x_{D-1}}{x_D}]
+
+    where :math:`D` is the index of the part used as common denominator.
+
+    Parameters
+    ----------
+    mat: numpy.ndarray
+       a matrix of proportions where
+       rows = compositions and
+       columns = components
+
+    denominator_idx: int
+       the index of the column (2D-matrix) or position (vector) of
+       `mat` which should be used as the reference composition
```

Comment:
Can we specify the default value here?  Something like

`default=0 to specify the first column`

Reply:
Done!